app.rb

```diff
@@ -1,8 +1,18 @@
 require 'bundler/setup'
 require 'sinatra'
 require 'sinatra/reloader'
+require 'pry'
 
 # Load specific routes
 require_relative 'controllers/welcome'
 require_relative 'controllers/apartments'
 require_relative 'controllers/tenants'
+
+require_relative 'db/connection.rb'
+require_relative 'models/apartment'
+require_relative 'models/tenant'
```

Comment:
These should be required before the controllers, so the models are available for use by the controllers.  No hoisting in ruby.